tests/zfs-tests/tests/functional/write_dirs/write_dirs_001_pos.ksh

```diff
@@ -48,6 +48,7 @@ verify_runnable "both"
 function cleanup
 {
 	destroy_dataset $TESTPOOL/$TESTFS
+	zpool sync $TESTPOOL # fence for free space accounting
```

Comment:
If the concern is waiting until everything has been freed, then can we instead use the `wait_freeing()` helper function for this.

Reply:
That does not work; neither does `zpool wait -t free`.

Reply:
Maybe doing both would be worthwhile though: wait for the pool to finish freeing and then wait for the accounting to be updated.

Reply:
It appears the Ubuntu 16.04 builder still failed in these tests because they took to long to fill the filesystem.

http://build.zfsonlinux.org/builders/Ubuntu%2016.04%20x86_64%20%28TEST%29/builds/11262/steps/shell_9/logs/summary

Reply:
We can have the tests use smaller file vdevs to make it easier to fill the fs before the timeout. I can include that in this PR if you like that idea?

Reply:
Actually maybe just using a single disk will be enough.

Reply:
Ah I see that when I eliminated partitioning in the setup for these tests I made it use all disks instead of one, making the pool bigger and more time consuming to fill. Will fix!